src/style.css

```diff
@@ -2,0 +1,92 @@
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir', Arial, sans-serif;
+}
+
+input {
+  all: unset;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
```

Comment:
Don't fix container size (if there is no such a requirement). In this case, setting both `width: 100%;` and `height: 100%;` on the input element might lead to unexpected behavior if the parent's size is not defined. It's usually better to control the size of an input with padding and possibly max-width or max-height.